### PR TITLE
Add usage documentation about maximum lag and retries

### PR DESCRIPTION
**Pull request by Xymph**
In all the work and discussion regarding #112, I took a shortcut in not documenting the feature, and my trusty reviewer didn't remind me about it either. :wink: 

The text already uses `WikimateException` which doesn't exist yet, but will be defined in my next PR (after this and #133 are merged).

### CODE REVIEW

**Review by waldyrious — APPROVED** [review]
LGTM :) I'll wait until this is rebased to merge it.

**Comment by waldyrious** [comment]
(I do have to mention that I feel a little uneasy with the time-traveling aspect of an earlier commit mentioning a concept that will be introduced in a later commit, but hey, nothing wrong with leaving a little mystery to the future code archeologists :P)

**Comment by Xymph** [comment]
> (I do have to mention that I feel a little uneasy with the time-traveling aspect of an earlier commit mentioning a concept that will be introduced in a later commit, but hey, nothing wrong with leaving a little mystery to the future code archeologists :P)

No offense intended, but would you have felt equally uneasy if I hadn't pointed it out explicitly?
Anyway, thanks for working with me, even into the future. :wink:

**Comment by waldyrious** [comment]
> No offense intended, but would you have felt equally uneasy if I hadn't pointed it out explicitly?

[Good question](https://en.wikipedia.org/wiki/If_a_tree_falls_in_a_forest)! I would likely not have noticed it — or rather, I'd assume it already existed in the codebase, which I don't know nearly as intimately as you do 🙂.

**Pull request merged**
